appveyor.yml

```
@@ -5,6 +5,19 @@ environment:
   FORK_USER: ocaml
   FORK_BRANCH: master
   CYG_ROOT: C:\cygwin64
```

```suggestion
  CYG_ROOT: C:\cygwin64
  OPAM_VERSION: 2.0.8
```
Could you try this? it might help the doubling of output in opam

---

I can reproduce in local with Opam 2.0.8, so this won't change it.

---

ah, I see. That sounds like an opam-on-windows™ bug indeed then :/ Have you tried with opam 2.1.0~beta4?